src/core/public/legacy/legacy_service.ts

```diff
@@ -96,25 +71,26 @@ export class LegacyPlatformService {
         linkToLastSubUrl: navLink.linkToLastSubUrl,
       })
     );
+  }
+
+  public start({ core, targetDomElement, plugins }: StartDeps) {
+    // Inject parts of the new platform into parts of the legacy platform
+    // so that legacy APIs/modules can mimic their new platform counterparts
+    require('ui/new_platform').__newPlatformStart__(core, plugins);
```

Comment:
I think since we don't need these magic exports anymore, we could just rename this to `start`.

Reply:
I'd be fine with `__start__`. I just want to make it clear that this is "special function" that shouldn't just be called from any code.